paradex_py/message/order.py

```diff
@@ -38,3 +38,45 @@ def build_order_message(chain_id: int, o: Order) -> TypedData:
         },
     }
     return message
+
+
+def build_order_message_for_modify(chain_id: int, o: Order) -> TypedData:
+    message = {
+        "domain": {"name": "Paradex", "chainId": hex(chain_id), "version": "1"},
+        "primaryType": "ModifyOrder",
+        "types": {
+            "StarkNetDomain": [
+                {"name": "name", "type": "felt"},
+                {"name": "chainId", "type": "felt"},
+                {"name": "version", "type": "felt"},
+            ],
+            "ModifyOrder": [
+                {
+                    "name": "timestamp",
+                    "type": "felt",
+                },  # Time of signature request in ms since epoch; Acts as a nonce;
+                {"name": "market", "type": "felt"},  # E.g.: "ETH-USD-PERP"
+                {"name": "side", "type": "felt"},  # Buy or Sell
+                {"name": "orderType", "type": "felt"},  # Limit or Market
```

Comment:
Only limit orders are allowed to be modified :) Should remove `Market` in the comment here